notebook/flaml_pytorch_cifar10.ipynb

```diff
@@ -301,9 +337,225 @@
+    },
+    {
+     "data": {
+      "text/html": [
+       "== Status ==<br>Memory usage on this node: 7.0/7.7 GiB: ***LOW MEMORY*** less than 10% of the memory on this node is available for use. This can cause unexpected crashes. Consider reducing the memory used by your application or reducing the Ray object store size by setting `object_store_memory` when calling `ray.init`.<br>Using AsyncHyperBand: num_stopped=0\n",
+       "Bracket: Iter 4.000: None | Iter 1.000: None<br>Resources requested: 0/2 CPUs, 0/0 GPUs, 0.0/0.9 GiB heap, 0.0/0.45 GiB objects<br>Result logdir: /home/ec2-user/FLAML/notebook/logs/train_cifar_2021-12-02_20-34-11<br>Number of trials: 1/50 (1 PENDING)<br><br>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-12-02 20:35:15,088\tWARNING tune.py:518 -- SIGINT received (e.g. via Ctrl+C), ending Ray Tune run. This will try to checkpoint the experiment state one last time. Press CTRL+C one more time (or send SIGINT/SIGKILL/SIGTERM) to skip. \n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "== Status ==<br>Memory usage on this node: 7.0/7.7 GiB: ***LOW MEMORY*** less than 10% of the memory on this node is available for use. This can cause unexpected crashes. Consider reducing the memory used by your application or reducing the Ray object store size by setting `object_store_memory` when calling `ray.init`.<br>Using AsyncHyperBand: num_stopped=0\n",
+       "Bracket: Iter 4.000: None | Iter 1.000: None<br>Resources requested: 0/2 CPUs, 0/0 GPUs, 0.0/0.9 GiB heap, 0.0/0.45 GiB objects<br>Result logdir: /home/ec2-user/FLAML/notebook/logs/train_cifar_2021-12-02_20-34-11<br>Number of trials: 1/50 (1 PENDING)<br><table>\n",
+       "<thead>\n",
+       "<tr><th>Trial name          </th><th>status  </th><th>loc  </th><th style=\"text-align: right;\">  batch_size</th><th style=\"text-align: right;\">  l1</th><th style=\"text-align: right;\">  l2</th><th style=\"text-align: right;\">         lr</th><th style=\"text-align: right;\">  num_epochs</th></tr>\n",
+       "</thead>\n",
+       "<tbody>\n",
+       "<tr><td>train_cifar_34fb4d64</td><td>PENDING </td><td>     </td><td style=\"text-align: right;\">           4</td><td style=\"text-align: right;\">   3</td><td style=\"text-align: right;\">   7</td><td style=\"text-align: right;\">0.000120469</td><td style=\"text-align: right;\">           1</td></tr>\n",
+       "</tbody>\n",
+       "</table><br><br>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-12-02 20:35:15,350\tERROR tune.py:557 -- Trials did not complete: [train_cifar_34fb4d64]\n",
```

Comment:
There is an error in execution.
Could you clear the output of this notebook? Like the original.